File: Aplikacija/GiveWaveAPI/GiveWaveAPI/Helpers/ProductHelper.cs
using GiveWaveAPI.Models;

namespace GiveWaveAPI.Helpers
{
    public class ProductHelper
    {
        public int Id { get; set; }
        public string Naziv { get; set; }
        public string Opis { get; set; }
        public string Mesto { get; set; }
        public string ImageUrl { get; set; }
        public Status status { get; set; }
        public string emailKorisnika{ get; set; }
        public string Kategorija { get; set; }
    }
}

File: Aplikacija/GiveWave/src/app/Models/ProductHelper.ts
export interface ProductHelper
{
    Naziv:string,
    Opis:String,
    Mesto:String,
    status:string,
    emailKorisnika:string|null,
    Kategorija:string,
    novaKategorija:string,
    parentKategorija:string
}

export enum Status{
    New,
    SecondHand
}